Dockerfile_multiarch

```diff
@@ -26,6 +26,16 @@ ARG BASE_IMAGE
 ARG BASE_IMAGE_VERSION
 FROM ${BASE_IMAGE}:${BASE_IMAGE_VERSION}
 
+WORKDIR /
+COPY --from=builder /workspace/manager .
+COPY --from=builder /workspace/daemon ./kruise-daemon
+
+RUN mkdir -p /log /tmp
```

Comment:
plz merge consecutive RUN into a single RUN statement to speed up image building